File: 04_ِDay 3 (intro to conditional statements)/solution.c
#include <stdio.h>
#include <string.h>


int main()
{

    int n;

    scanf("%d",&n);

    if( n%2 != 0){
        printf("Weird");
    }
    else{
        if(n <= 5)
            printf("Not Weird");
        else if ( n <= 20 )
            printf("Weird");
        else {
            printf("Not Weird");
        }
    }

}

File: 04_ِDay 3 (intro to conditional statements)/solution.py
import math
import os
import random
import re
import sys

#input
n = int(input())

#if else statements and output

if n%2 != 0:
    print('Weird')
else:
    if n<=5:
        print('Not Weird')
    elif n<=20:
        print('Weird')
    else:
        print('Not Weird')
